vision/unit_tests/test_annotations.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+
+class TestAnnotations(unittest.TestCase):
+    @staticmethod
+    def _get_target_class():
+        from google.cloud.vision.annotations import Annotations
+
+        return Annotations
+
+    def _make_one(self, *args, **kw):
+        return self._get_target_class()(*args, **kw)
+
+    def test_ctor(self):
+        annotations = self._make_one(
+            faces=[True], properties=[True], labels=[True], landmarks=[True],
+            logos=[True], safe_searches=[True], texts=[True])
+        self.assertTrue(annotations.faces[0])
+        self.assertTrue(annotations.properties[0])
+        self.assertTrue(annotations.labels[0])
+        self.assertTrue(annotations.landmarks[0])
+        self.assertTrue(annotations.logos[0])
+        self.assertTrue(annotations.safe_searches[0])
+        self.assertTrue(annotations.texts[0])
+
+    def test_from_pb(self):
+        from google.cloud.grpc.vision.v1 import image_annotator_pb2
+
+        image_response = image_annotator_pb2.AnnotateImageResponse()
+        annotations = self._make_one().from_pb(image_response)
+        self.assertEquals(annotations.labels, [])
```

Comment:
`assertEquals` is a [deprecated alias][1] for `assertEqual`

[1]: https://docs.python.org/3.6/library/unittest.html#deprecated-aliases